modules/light-clients/08-wasm/testing/wasm_endpoint.go

```diff
@@ -19,10 +19,18 @@ type WasmEndpoint struct {
 
 var (
 	// Represents the code of the wasm contract used in the tests with a mock vm.
-	Code                   = []byte("01234567012345670123456701234567")
-	contractClientState    = []byte{1}
-	contractConsensusState = []byte{2}
-	ErrMockContract        = errors.New("mock contract error")
+	Code                              = []byte("01234567012345670123456701234567")
+	contractClientState               = []byte{1}
+	contractConsensusState            = []byte{2}
+	ErrMockContract                   = errors.New("mock contract error")
+	MockClientStateBz                 = []byte("client-state-data")
+	MockConsensusStateBz              = []byte("consensus-state-data")
+	MockValidProofBz                  = []byte("valid proof")
+	MockInvalidProofBz                = []byte("invalid proof")
+	MockInvalidClientStateProofBz     = []byte("invalid client state proof")
```

Comment:
sorry for all the nits i think the this is the last one 🙈 , but i _think_ it might be enough just to use the MockValidProofBz and MockInvalidProofBz for the client state/consensus state proof bz, since these are all mocks anyway.

Reply:
no problem